src/parquet/file/writer.h

```diff
@@ -103,7 +106,8 @@ class ParquetFileWriter {
 
   static std::unique_ptr<ParquetFileWriter> Open(std::shared_ptr<OutputStream> sink,
       std::shared_ptr<schema::GroupNode>& schema,
-      MemoryAllocator* allocator = default_allocator());
+      MemoryAllocator* allocator = default_allocator(),
+      std::shared_ptr<WriterProperties> properties = default_writer_properties());
```

Comment:
I guess we could benefit from some const refs in other parameters in this function